apps/frontend/src/app/5_pages/MarketMakingPage/components/BobClaimFeesModal/hooks/useHandleSubmit.ts

```diff
@@ -0,0 +1,105 @@
+import { useCallback } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Transaction,
+  TransactionType,
+} from '../../../../../3_organisms/TransactionStepDialog/TransactionStepDialog.types';
+import { GAS_LIMIT } from '../../../../../../constants/gasLimits';
+import { useCrocContext } from '../../../../../../contexts/CrocContext';
+import { useTransactionContext } from '../../../../../../contexts/TransactionContext';
+import { useAccount } from '../../../../../../hooks/useAccount';
+import { useCurrentChain } from '../../../../../../hooks/useChainStore';
+import { translations } from '../../../../../../locales/i18n';
+import { PoolPositionType } from '../../../MarketMakingPage.types';
+import { useGetPool } from '../../../hooks/useGetPool';
+import { AmbientPosition } from '../../AmbientMarketMaking/AmbientMarketMaking.types';
+import { AmbientLiquidityPool } from '../../AmbientMarketMaking/utils/AmbientLiquidityPool';
+import { AmbientLiquidityPoolDictionary } from '../../AmbientMarketMaking/utils/AmbientLiquidityPoolDictionary';
+import { DEFAULT_SLIPPAGE } from '../../BobDepositModal/BobDepositModal.constants';
+
+export const useHandleSubmit = (
+  pool: AmbientLiquidityPool,
+  position: AmbientPosition,
+  onComplete: () => void,
+) => {
+  const { signer } = useAccount();
+  const { croc } = useCrocContext();
```

Comment:
```suggestion
```
`signer` is not used anywhere (it's only a dependency in a callback) and you don't need `croc`.